### PR TITLE
Release 0.3

### PR DESCRIPTION
**Pull request by nsheff**
adds `divvy write` and `divvy list` commands, fixes yaml warnings, reduces verbosity.

- [x] version bump
- [x] changelog

### CODE REVIEW

File: divvy/compute.py [hunk 1]
```diff
@@ -220,15 +223,20 @@ def update_packages(self, config_file):
...
             # Any compute.submission_template variables should be made
             # absolute, relative to current divvy configuration file.
-            if "compute" in env_settings:
-                _LOGGER.warning("In your divvy config file, please use 'compute_packages' instead of 'compute'")
-                env_settings["compute_packages"] = env_settings["compute"]
+            if old_compute_key in env_settings:
+                warnings.warn("Divvy compute configuration '{}' section changed "
+                              "to '{}'".format(old_compute_key, new_compute_key),
+                              DeprecationWarning)
+                env_settings[new_compute_key] = env_settings[old_compute_key]
```
Comment (vreuter) [inline divvy/compute.py]: This way of handling the deprecation/backcompat allows the keys to map to different values.

File: divvy/utils.py [hunk 2]
```diff
@@ -296,6 +282,27 @@ def is_url(maybe_url):
...
+        env_settings = yaml.load(f, yaml.SafeLoader)
+    _LOGGER.debug("Parsed environment settings: %s",
+                  str(env_settings))
+    # Any compute.submission_template variables should be made
+    # absolute, relative to current divvy configuration file.
+    if OLD_COMPUTE_KEY in env_settings:
+        warnings.warn("Divvy compute configuration '{}' section changed "
+                      "to '{}'".format(OLD_COMPUTE_KEY, NEW_COMPUTE_KEY),
+                      DeprecationWarning)
+        env_settings[NEW_COMPUTE_KEY] = env_settings[OLD_COMPUTE_KEY]
```
Comment (vreuter) [inline divvy/utils.py]: It bothers me that this still allows subsequent mutations to have the keys get out of sync, but this is used in small enough fashion that I don't think it's a big deal. This sort of thing motivates https://github.com/pepkit/attmap/issues/20.
Same as https://github.com/pepkit/divvy/pull/28#discussion_r274067111

**Comment by vreuter** [comment]
@nsheff I think this is set for merge now if you're ready

**Comment by nsheff** [comment]
great thanks!

**Pull request merged**